### ISSUE TITLE
Request: Select slot for Tesco

### ISSUE THREAD

**Issue by eperdeme** [paulmaunders/delivery-slot-bot#67]
Howdi, 

I'm sure I saw an issue way back, but did you ever get round to implementing slot reservation for Tesco home delivery ?

**Comment by obbardc** [comment]
i'm not sure how complex tesco home delivery is; but for collection at least you have to add an item to basket and checkout which involves entering card pin number & authenticating the card transaction using verifided by visa etc.

is there a way around all that and just booking a slot at all?

**Comment by CharlesButcher** [comment]
For home delivery you can book a slot for, I think, two hours before you have to put something in the trolley and pay for it. The ability to reserve a slot would be cool because it would save a lot of panicked scrabbling on being woken by Pushover at 6 am.

My wife makes me wash all our shopping before putting it away. If the bot could do this it would be a bonus.

**Comment by obbardc** [comment]
ok i will try that next time i checkout, and attempt to save the html, but only once a slot appears ;-)

only on click and collect here though

**Comment by RossBaptie** [comment]
yeah click and collect is the same - you can 'reserve' the slot immediately without any need to put items in basket or checkout, but you must checkout within 2 hours of that to retain it

**Comment by eperdeme** [comment]
Yes, thats the feature I thought would be possible to do. I've seen it done in python code of similar functions to this, it's really just issuing a click to the box where the time is, which I think we alreadyk now as we report the delivery slot times.
